.github/workflows/go.yml

```diff
@@ -0,0 +1,34 @@
+name: Go
```

Comment:
Is this ci file conflict with the one we already have in master?

Reply:
@hangyan This one creates and stores the artifacts the other one did not exist when I created this one. As far as I can see the tests are executed there.